packages/core/admin/admin/src/components/ConfigurationsProvider/index.js

```diff
@@ -0,0 +1,47 @@
+import React, { useReducer } from 'react';
+import PropTypes from 'prop-types';
+import { ConfigurationsContext } from '../../contexts';
+import reducer, { initialState } from './reducer';
+
+const ConfigurationsProvider = ({
+  children,
+  authLogo,
+  menuLogo: defaultMenuLogo,
+  showReleaseNotification,
+  showTutorials,
+}) => {
+  const [{ customMenuLogo }, dispatch] = useReducer(reducer, initialState);
```

Comment:
You should add the init function to the useReducer hook in order to populate the state with the default values.

Reply:
```js
const [{ logos: { auth, menu }  }, dispatch] = useReducer(reducer, initialState);
```